packages/coinstac-ui/app/render/state/ducks/consortia.js

```diff
@@ -104,6 +105,134 @@ export const setActiveComputation = applyAsyncLoading(
 );
 
 /**
+ * Set inputs on a consortium's selected computation.
```

Comment:
:boom: Document design discussion here. We should use these ideas when expanding the idea of “inputs” in COINSTAC. (See #12)